qi-lib/flow/core/passes.rkt

```diff
@@ -0,0 +1,53 @@
+#lang racket/base
+
+(require (for-syntax racket/base
+                     (for-syntax racket/base racket/syntax)
+                     macro-debugger/emit))
+
+(provide (for-syntax define-pass
+                     run-passes))
+
+(begin-for-syntax
```

Comment:
Can we define this module at phase 0 and then `require` at the appropriate phase in client modules?